src/ts/components/uicontainer.ts

```diff
@@ -158,7 +158,7 @@ export class UIContainer extends Container<UIContainerConfig> {
       },
     }, {
       name: 'focusin',
-      handler: (e) => {
+      handler: () => {
```

Comment:
This change is completely unrelated to this PR.

Reply:
It is I agree but the argument is unused in the method (a bit of code cleaning doesn't hurt in the long run)

Reply:
Code cleaning, when if unrelated to the PR, should be done in a separate PR.